nasa/harmony
# HARMONY-1173: Fix shapefile subsetting section of intro notebook and other general issues

## Jira Issue ID
HARMONY-1173, HARMONY-942, HARMONY-1031

## Description
I made a bunch of changes to the intro notebook to update collections being used, granules being used, time ranges being used, authentication issues when not using a .netrc file, and getting the shapefile subsetting section of the notebook to run.

I had issues installing and using GDAL locally so I also added a Dockerfile to build a docker image specifically to run the notebooks. This docker image should contain all libraries needed to execute the notebooks.

## Local Test Steps
1. `npm run build-notebooks-image`
2. `npm run run-notebooks-image`
3. Open the link to the Jupyter notebook printed out in the terminal
4. Verify you can run everything up until the WMS cell (WMS is still broken, but the shapefile section above it should work). Note that you will need to provide your EDL username and password in UAT when prompted since the docker image will not have your .netrc file.

## PR Acceptance Checklist
